pkg/gardenlet/controller/seed/seed_control.go

```diff
@@ -414,3 +421,24 @@ func deleteBackupBucketInGarden(ctx context.Context, k8sGardenClient client.Clie
 
 	return client.IgnoreNotFound(k8sGardenClient.Delete(ctx, backupBucket))
 }
+
+// CleanupLegacyPriorityClasses deletes reversed-vpn-auth-server, fluent-bit, loki, istio-ingressgateway and istiod priority classes
+func CleanupLegacyPriorityClasses(ctx context.Context, seedClient client.Client) error {
+	var fns []flow.TaskFn
+	legacyPriorityClasses := sets.NewString([]string{"reversed-vpn-auth-server", "fluent-bit", "loki", "istio-ingressgateway", "istiod"}...)
```

Comment:
You don't need `"reversed-vpn-auth-server", "istio-ingressgateway", "istiod"` here since those `PriorityClass`es are managed by GRM/`ManagedResource`s and will be cleaned up automatically.

Reply:
Hmm `reversed-vpn-auth-server` is not managed by a `ManagedResource`: https://github.com/gardener/gardener/blob/90ac648f7d28287819b1780ab66cb1cbab25e14e/pkg/operation/botanist/component/vpnauthzserver/external_authz_server.go#L83-L90